code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -134,6 +152,22 @@
 		var/datum/action/A=X
 		A.UpdateButtonIcon()
 
+/obj/item/clothing/head/helmet/space/plasmaman/AltClick(mob/user)
+	..()
+	. = COMSIG_CLICK_ALT
+	if(!attached_hat)
+		return
+	if(user.put_in_active_hand(attached_hat))
+		to_chat(user, "<span class='notice'>You removed [attached_hat.name] from the helmet!</span>")
+		attached_hat = null
+		update_icon()
+	else if(user.put_in_inactive_hand(attached_hat))
+		to_chat(user, "<span class='notice'>You removed [attached_hat.name] from the helmet!</span>")
+		attached_hat = null
+		update_icon()
+	else
+		to_chat(user, "<span class='notice'>You can't remove [attached_hat.name] from the helmet because your hands are full!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>You can't remove the [attached_hat.name] from the helmet because your hands are full!</span>")
```